### PR TITLE
fix: show plus member since

### PR DESCRIPTION
**Pull request by omBratteng**
## Changes

The tooltip value was broken.
See also https://github.com/dailydotdev/daily-api/pull/2440

<table>
 <tr>
  <td>Before
  <td>After
 <tr>
  <td>
  <td>
</table>

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### CODE REVIEW

File: packages/shared/src/components/widgets/PostUsersHighlights.tsx [hunk 1]
```diff
@@ -127,6 +128,7 @@ export const UserHighlight = (props: UserHighlightProps): ReactElement => {
     ('handle' in user ? user.handle : user.username) || id;
   const reputation = 'reputation' in user ? user.reputation : NaN;
   const companies = 'companies' in user ? user.companies : [];
+  const isPlus = 'isPlus' in user ? user.isPlus : false;
```
Comment (rebelchris) [inline packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: Why do we do it so weird here vs:
```const isPlus = user?.isPlus;```

**Review by rebelchris — APPROVED** [review]
One non blocking comment

File: packages/shared/src/components/widgets/PostUsersHighlights.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: image
Strange type errors

Comment (omBratteng) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: Ah, it's because `SourceAuthorProps` also has `Source` as a type :triggered:

Comment (rebelchris) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: Rather check with typeof maybe? (just feels more verbose?)

Comment (omBratteng) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: Not sure if it's possible, typescript will complain as long as it thinks that it doesn't exist on the type.

Comment (omBratteng) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: image

example

Comment (omBratteng) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: Alterative is this
image

Comment (rebelchris) [reply packages/shared/src/components/widgets/PostUsersHighlights.tsx:131]: Ah let's leave it then, not a big deal

**Pull request merged**